### PR TITLE
Add UI for media view

### PR DESCRIPTION
**Pull request by lucyxubroad**
### Overview 
Add UI for media view and comments view controller

### Changes Made
Created a custom slider
Created the comments view controller
Laid out reviews and thoughts section of media view controller

### Related Issues
#22  #21 

### Screenshots/GIFs

Screen-Recording-2020-08-01-at-1

### CODE REVIEW

File: Flick/Assets.xcassets/rw-logo.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Razewarelogo_1024.png",
```
Comment (haiyingweng) [inline Flick/Assets.xcassets/rw-logo.imageset/Contents.json:4]: wait what is this for lol

File: Flick/Controllers/MediaCardViewController.swift [hunk 2]
```diff
@@ -11,17 +11,53 @@ import UIKit
...
+
+  override var contentSize:CGSize {
+      didSet {
+          self.invalidateIntrinsicContentSize()
+      }
+  }
+
+  override var intrinsicContentSize: CGSize {
+      self.layoutIfNeeded()
+    return CGSize(width: UIView.noIntrinsicMetric, height: contentSize.height)
```
Comment (haiyingweng) [inline Flick/Controllers/MediaCardViewController.swift:24]: spacing is a little off here

File: Flick/Controllers/MediaCardViewController.swift [hunk 3]
```diff
@@ -30,36 +66,247 @@ class MediaCardViewController: UIViewController {
...
+        scrollView.bounces = false
+        view.addSubview(scrollView)
+
+        scrollView.addSubview(contentView)
+        contentView.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height)
 
         summaryView = MediaSummaryView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: 0))
         summaryView.sizeToFit()
-        view.addSubview(summaryView)
+        contentView.addSubview(summaryView)
```
Comment (haiyingweng) [inline Flick/Controllers/MediaCardViewController.swift:84]: this `contentView` kinda confuses me cell's `contentView` lol

File: Flick/Controls/RatingsSliderControl.swift [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+//
+
+import UIKit
+
+class SliderControlTrackLayer: CALayer {
+  weak var ratingsSliderControl: RatingSliderControl?
+
+  override func draw(in context: CGContext) {
+    guard let slider = ratingsSliderControl else {
+      return
```
Comment (haiyingweng) [inline Flick/Controls/RatingsSliderControl.swift:16]: something off with spacing here as well

File: Flick/Controls/RatingsSliderControl.swift [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+    private let trackLayer = SliderControlTrackLayer()
+
+    // MARK: - Private Data Vars
+    private let iconSize = CGSize(width: 14, height: 14)
+    private let minimumValue: CGFloat = 0
+    private let maximumValue: CGFloat = 1
+    private var previousLocation = CGPoint()
+    private let personalIconSize = CGSize(width: 20, height: 20)
+
+    // MARK: - Publc Data Vars
```
Comment (haiyingweng) [inline Flick/Controls/RatingsSliderControl.swift:53]: Public?

File: Flick/Assets.xcassets/rw-logo.imageset/Contents.json (hunk 1 above)
Comment (lucyxubroad) [reply Flick/Assets.xcassets/rw-logo.imageset/Contents.json:4]: oh lol sorry that was added by mistake

**Pull request merged**